File: src/com/sterndu/util/exceptions/ClosedMemoryLocationException.java
package com.sterndu.util.exceptions;

public class ClosedMemoryLocationException extends Exception {

	/**
	 *
	 */
	private static final long serialVersionUID = 4810813237605305525L;

	public ClosedMemoryLocationException() {}

	public ClosedMemoryLocationException(String message) {
		super(message);
	}

	public ClosedMemoryLocationException(String message, Throwable cause) {
		super(message, cause);
	}

	public ClosedMemoryLocationException(String message, Throwable cause, boolean enableSuppression,
			boolean writableStackTrace) {
		super(message, cause, enableSuppression, writableStackTrace);
	}

	public ClosedMemoryLocationException(Throwable cause) {
		super(cause);
	}



}

File: src/com/sterndu/util/exceptions/ClosedMemoryLocationException.kt
@file:JvmName("ClosedMemoryLocationException")
package com.sterndu.util.exceptions

class ClosedMemoryLocationException : Exception {
	constructor()
	constructor(message: String?) : super(message)
	constructor(message: String?, cause: Throwable?) : super(message, cause)
	constructor(
		message: String?, cause: Throwable?, enableSuppression: Boolean,
		writableStackTrace: Boolean
	) : super(message, cause, enableSuppression, writableStackTrace)

	constructor(cause: Throwable?) : super(cause)
}
